ui/raidboss/data/05-shb/raid/e12n.js

```diff
@@ -1,8 +1,65 @@
+import Conditions from '../../../../../resources/conditions.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import { Responses } from '../../../../../resources/responses.js';
 import ZoneId from '../../../../../resources/zone_id.js';
-
 export default {
   zoneId: ZoneId.EdensPromiseEternity,
   timelineFile: 'e12n.txt',
   triggers: [
+    {
+      id: 'E12N Maleficium',
+      netRegex: NetRegexes.startsUsing({ id: '5872', source: 'Eden\'s Promise', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E12N Formless Judgment',
+      netRegex: NetRegexes.startsUsing({ id: '5873', source: 'Eden\'s Promise' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankCleave(),
+    },
+    {
+      id: 'E12N Boulders Impact',
+      netRegex: NetRegexes.ability({ id: '586E', source: 'Titanic Bomb Boulder', capture: false }),
+      suppressSeconds: 5,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Between small boulders',
```

Comment:
How do you feel about bombs vs boulders?
```suggestion
          en: 'Between small bombs',
```

Reply:
I don't care, and you're right, that's probably more consistent with other triggers.